.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules
-.DS_Store
```

Comment:
Your insistence upon `.DS_Store` not belonging in the global `.gitignore` is perhaps my favorite of your windmills.